nipype/algorithms/confounds.py

```diff
@@ -236,6 +252,14 @@ class FramewiseDisplacementInputSpec(BaseInterfaceInputSpec):
         "NIPY",
         desc="Source of movement parameters",
         mandatory=True)
+    metric = traits.Enum(
+        'L1',
+        'riemannian',
+        usedefault=True,
+        desc='Distance metric to apply: '
+        'L1 = Manhattan distance (original definition),'
```

Comment:
I think a semi-colon will make this slightly more readable than a comma. And when using multiple lines, put a space at the end of each line. This string is currently equivalent to `'Distance metric to apply: L1 = Manhattan distance (original definition),riemannian = Riemannian distance on theSpecial Euclidean group in 3D (geodesic)`.